pkg/config/types/generated_constants.go

```diff
@@ -47,6 +47,7 @@ const ComputeAllocatedCapacityGPUKey = "compute.allocatedcapacity.gpu"
 const ComputeAllowListedLocalPathsKey = "compute.allowlistedlocalpaths"
 const WebUIEnabledKey = "webui.enabled"
 const WebUIListenKey = "webui.listen"
+const WebUIBackendKey = "webui.backend"
```

Comment:
Isn't listen and backend same ?

Reply:
no listen is the address the webserver is listening to, but the frontend directly calls the orchestrator node (backend) and needs to know its address

the right thing is to have the webui webserver call the orchestrator and not the frontend (javascript), but that is a bigger change and unfortunately how the legacy webui used to work